### PR TITLE
Fixes for oidc-uma-sample

### PR DESCRIPTION
**Pull request by Kayathiri-Mahendrakumaran**

### CODE REVIEW

**Review by ashendes — COMMENTED** [review]
The keystore for all the other samples is maintained in `etc/resources/`. Shall we update the keystore there and change the build plugin for the UMA samples to pull the keystore from that directory? To centralize the keystore maintenance.
@janakamarasena WDYT?

**Review by omindu — APPROVED** [review]

**Comment by janakamarasena** [comment]
@ashendes +1 for the suggestion.

File: oidc-uma-samples/photo-edit/src/main/resources/appone.properties [hunk 1]
```diff
@@ -11,7 +11,7 @@ authzEndpoint=https://localhost:9443/oauth2/authorize
 OIDC_LOGOUT_ENDPOINT=https://localhost:9443/oidc/logout
 sessionIFrameEndpoint=https://localhost:9443/oidc/checksession
 tokenEndpoint=https://localhost:9443/oauth2/token
-post_logout_redirect_uri=http://localhost.com:8080/pickup-dispatch/oauth2client
+post_logout_redirect_uri=http://localhost.com:8080/photo-edit/oauth2client
 api_endpoint=http://localhost:39090/bookings
 idp_url=https://localhost:9443
 resource_id=
```
Comment (ashendes) [inline oidc-uma-samples/photo-edit/src/main/resources/appone.properties:17]: Lets add a new line at the end.

Comment (Kayathiri-Mahendrakumaran) [reply oidc-uma-samples/photo-edit/src/main/resources/appone.properties]: addressed with c6ecf133e17d127adb2d49c0c9bb5d46664d1941

**Review by ashendes — APPROVED** [review]

**Pull request merged**